lib/ace/ext/papeeria/latex_behaviour.coffee

```diff
@@ -1,9 +1,26 @@
 define((require, exports, module) ->
   Behaviour = require("ace/mode/behaviour").Behaviour
+  COMMENT_REGEX = /comment/
+  ESCAPE_REGEX = /escape/
+  MATH_REGEX = /math/
+  EQUATION_REGEX = /equation/
+  RPAREN_REGEX = /rparen/
+  correspondingClosing = {
+      '(': ')',
+      '[': ']'
+      '{': '}'
+  }
+
+
+  isCommentToken = (token, column) -> token? and COMMENT_REGEX.test(token.type) and column != 0
```

Comment:
It is. If we're on the start of a line, which starts with a comment (like this `|%a comment`), current token has a type `comment`, but it's alright to insert closing dollars or brackets.
Moved it to the start.